docs/multicluster/user-guide.md

```diff
@@ -312,57 +334,123 @@ Then create below `ClusterClaim` and `ClusterSet` in the cluster `test-cluster-n
 apiVersion: multicluster.crd.antrea.io/v1alpha1
 kind: ClusterClaim
 metadata:
-  name: north-membercluster-id
+  name: test-cluster-north
   namespace: kube-system
 name: id.k8s.io
 value: test-cluster-north
 ---
 apiVersion: multicluster.crd.antrea.io/v1alpha1
 kind: ClusterClaim
 metadata:
-  name: clusterset-id
+  name: test-clusterset
   namespace: kube-system
 name: clusterSet.k8s.io
 value: test-clusterset
 ---
 apiVersion: multicluster.crd.antrea.io/v1alpha1
 kind: ClusterSet
 metadata:
-    name: test-clusterset
-    namespace: kube-system
+  name: test-clusterset
+  namespace: kube-system
 spec:
-    leaders:
-      - clusterID: test-cluster-north
-        secret: "member-north-access-token"
-        server: "https://172.18.0.2:6443"
-    members:
-      - clusterID: test-cluster-north
-    namespace: antrea-mcs-ns
+  leaders:
+    - clusterID: test-cluster-north
+      secret: "member-north-access-token"
+      server: "https://172.18.0.2:6443"
+  members:
+    - clusterID: test-cluster-north
+  namespace: antrea-multicluster
 ```
 
 Then update the ClusterSet `test-clusterset` definition as below to include itself
-as a member cluster in the Namespace `antrea-mcs-ns` where the leader controller is
+as a member cluster in the Namespace `antrea-multicluster` where the leader controller is
 running in the cluster `test-cluster-north`.
 
 ```yaml
 apiVersion: multicluster.crd.antrea.io/v1alpha1
 kind: ClusterSet
 metadata:
-    name: test-clusterset
-    namespace: antrea-mcs-ns
+  name: test-clusterset
+  namespace: antrea-multicluster
 spec:
-    leaders:
-      - clusterID: test-cluster-north
-    members:
-      - clusterID: test-cluster-east
-        serviceAccount: "member-east-access-sa"
-      - clusterID: test-cluster-west
-        serviceAccount: "member-west-access-sa"
-      - clusterID: test-cluster-north
-        serviceAccount: "member-north-access-sa"
-    namespace: antrea-mcs-ns
+  leaders:
+    - clusterID: test-cluster-north
+  members:
+    - clusterID: test-cluster-east
+      serviceAccount: "member-east-access-sa"
+    - clusterID: test-cluster-west
+      serviceAccount: "member-west-access-sa"
+    - clusterID: test-cluster-north
+      serviceAccount: "member-north-access-sa"
+  namespace: antrea-multicluster
 ```
 
+## Multi-cluster Gateway Configuration
```

Comment:
I moved it here to be before the feature consumption sections. But have not review and update the section yet.